File: web2py/matlab/functions/TIDELV.m
function [tidelv]=TIDELV(ng,t,ampl,alpha,fndcst,acst)

deg2rad=pi/180;
tidelv=0;
for nc=1:length(ampl)
    arg=(acst(nc)*t+alpha(nc,ng))*deg2rad;
    tidelv=tidelv+fndcst(nc)*ampl(nc,ng)*cos(arg);
end

File: python/functions/TIDELV.py
import math

def TIDELV(ng, t, ampl, alpha, fndcst, acst):
    deg2rad = math.pi/180.0
    tidelv = 0.0

    for nc in range(len(ampl)):
        arg = (acst[nc]*t + alpha[nc][ng - 1])*deg2rad
        tidelv = tidelv + fndcst[nc]*ampl[nc]*math.cos(arg)

    return tidelv
